qt_binder/binder.py

```diff
@@ -568,6 +564,100 @@ def _on_trait_change(self, handler, name=None, remove=False,
                 for func, name in connectors:
                     func(self, name)
 
+    def _collect_renamings(self, binder_class):
+        """ Collect all of the renamings requested by Rename traits.
+        """
+        renamings = {}
+        # Find all requested renamings.
+        for name in dir(binder_class):
+            obj = getattr(binder_class, name)
+            if isinstance(obj, Rename):
+                renamings[obj.qt_name] = name
+        return renamings
+
+    def _add_qt_properties(self, binder_class, renamings, seen):
+        """ Add QtProperties.
+        """
+        meta_object = binder_class.qclass.staticMetaObject
+        for i in range(meta_object.propertyCount()):
+            meta_prop = meta_object.property(i)
+            qname = _python_name_for_qt_name(meta_prop.name())
+            name = renamings.get(qname, qname)
+            if name not in seen:
+                if name.endswith('_'):
+                    # See #21
+                    continue
+                binder_class.add_class_trait(name, QtProperty(meta_prop))
+                seen.add(name)
+
+    def _add_qt_signals_slots(self, binder_class, renamings, seen):
+        """ Add QtSignals and QtSlots.
+        """
```

Comment:
This method refactors and modifies/adds to the logic in lines 602-633 on master.